pkg/server/registry/service.go

```diff
@@ -17,18 +17,25 @@ import (
 type LocalContainerRegistryConfig struct {
 	DataPath string
 	Port     uint32
+	Image    string
 }
 
 func NewLocalContainerRegistry(config *LocalContainerRegistryConfig) *LocalContainerRegistry {
+	registryImage := config.Image
+	if registryImage == "" {
```

Comment:
I suggest removing this.

I understand why you did this, but this commit should be a breaking change and that's fine. We will inform the users to add `registry:2.8.3` after updating.

Please just add a `BREAKING CHANGE: ....` footer to the commit (above the sign off) and add it to the PR description.